backend/app/modules/modules_factory/pending.py

```diff
@@ -151,4 +153,5 @@ def create(chain_id: int) -> "PendingServices":
             project_rewards_service=project_rewards,
             multisig_signatures_service=multisig_signatures,
             projects_metadata_service=StaticProjectsMetadataService(),
+            uniqueness_quotients=uniqueness_quotients,
```

Comment:
what's the point in moving it here? if we do so to the PendingServices directly, do we need to store it in the pending_user_allocations?

Reply:
resolved in internal discussion